sites/next.skeleton.dev/src/content/docs/guides/mode.mdx

```diff
@@ -38,4 +43,20 @@ Dark Mode: Black Background
 
 ## Using Color Parings
 
-Skeleton provides opinionated utility classes called [Color Pairings](/docs/design/colors#pairings) to assist with balancing theme colors for light and dark mode.
+Skeleton provides utility classes called [Color Pairings](/docs/design/colors#pairings). These combine both base and dark mode classes into a singlular class, enabling an opinionated way to balance Skeleton theme colors for either mode.
+
+---
+
+## Lightswitch Alternatives
+
+In legacy versions, Skeleton provided a dedicated `<Lightswitch>` component for toggling dark mode using the `selector` strategy. Unfortunately this is no longer possible given the sheer number of permutations available:
+
+- Supporting the `selector`, `media`, or a hybrid strategy.
+- Supporting the unique APIs of each tech stacks and/or meta-framework.
+- Handling state and persistence; local vs remote storage (e.g. account-based storage)
+
+Given this, we recommend the following alternatives:
+
+1. Drop manual selection in favor of only the `media` strategy, to adhere to your user's operating system settings.
```

Comment:
1. "Forego user-based selection in favor..."
2. "Generate your own version based on our simple Astro component."